### PR TITLE
add explanation for ist0118 (PortNameIsNotUnderNamingConvention)

### PR DESCRIPTION
**Pull request by yxue**
https://github.com/istio/istio/pull/19375

### CODE REVIEW

File: content/en/docs/reference/config/analysis/ist0118/index.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+title: PortNameIsNotUnderNamingConvention
+layout: analysis-message
+---
+
+This message occurs when the port doesn't follow [Istio service port naming convention](/docs/ops/deployment/requirements/)
```
Comment (ericvn) [inline content/en/docs/reference/config/analysis/ist0118/index.md]: doesn't follow **_the_**

Comment (yxue) [reply content/en/docs/reference/config/analysis/ist0118/index.md]: done

**Comment by yxue** [comment]
@istio/wg-docs-maintainers-english could you please help to review the PR? Thanks!

File: content/en/docs/reference/config/analysis/ist0118/index.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
+title: PortNameIsNotUnderNamingConvention
+layout: analysis-message
+---
+
+This message occurs when the port doesn't follow the [Istio service port naming convention](/docs/ops/deployment/requirements/)
+or the port is unnamed. If you are not on the way to hardening your cluster, you can ignore the message.
```
Comment (ericvn) [inline content/en/docs/reference/config/analysis/ist0118/index.md]: Not the grammar person, but `If you are not on the way to hardening your cluster` doesn't seem normal to me. Also, would people who aren't trying to harden their clusters possibly still care about this message?

Comment (yxue) [reply content/en/docs/reference/config/analysis/ist0118/index.md]: Ok, I am going to remote that statement, WDYT?

**Pull request merged**